etc/profile-m-z/teams.profile

```diff
@@ -9,6 +9,8 @@ include teams.local
 # added by included profile
 #include globals.local
 
+# see #3404
+ignore apparmor
```

Comment:
there is a comment above about "ignore apparmor". can/should the comment be removed?

Reply:
You mean `# see #3404`? I added it for reference, we can remove it if it distract someone.

Reply:
no, i mean at the top of the file (line 5). :-)  Github didn't let me comment there.

Reply:
Uhh that what you mean. Yes it should be removed.